core/module/Dog/dog_modules/dog_module/Shadowlamb/core/SR_Player.php

```diff
@@ -2083,7 +2083,19 @@ public function giveItem(SR_Item $item)
 
 	public function itemAmountChanged(SR_Item $item, $amount_change, $modify=true)
 	{
-		$this->sr4_inventory->itemAmountChanged($item, $amount_change);
+		if ($this->sr4_inventory->contains($item))
+		{
+			$this->sr4_inventory->itemAmountChanged($item, $amount_change);
+		}
+		elseif ($this->sr4_mount_inv->contains($item))
+		{
+			$this->sr4_mount_inv->itemAmountChanged($item, $amount_change);
+		}
+		else
+		{
+			// TODO gizmore print error message (tehron)
```

Comment:
`GWF_Mail::sendMailS(GWF_BOT_EMAIL, 'gizmore@wechall.net', '[Shadowlamb] TODO gizmore print error message (tehron)', '-nt-');`

:stuck_out_tongue:

Reply:
Anyway, should not happen, so I'm not against throwing an exception here. Only way anyone is going to notice, probably. (And prevents users from using it as an infinite source of items.)

Reply:
resolved in 551da4d